Update README.vi.md (#20)
prd-khoa-luu/arizona-stay: README.vi.md

* Tàu điện ngầm Valley Metro (và xe buýt)
  * <img src="https://github.com/user-attachments/assets/9190d243-f7d3-4d41-9511-a048a90c87fd" width="25%">
  * Tàu điện nhẹ kết nối trung tâm Phoenix, Tempe, và Mesa -- hoàn hảo để khám phá
  * Tải ứng dụng Valley Metro, thêm thẻ tín dụng của bạn, và nạp 4 đô la, bao gồm các chuyến đi không giới hạn cả ngày
  * Lưu ý: luôn quét thẻ trước khi lên tàu; việc kiểm tra vé vẫn diễn ra
* Máy bay
  * <img src="https://github.com/user-attachments/assets/8920c027-5bee-4ad1-bfa0-b0634acda73a" width="25%">
  * Để bay đến các tiểu bang khác
  * Đặt vé: tìm chuyến bay trên Google Flights (https://www.google.com/travel/flights), sau đó đặt trực tiếp qua trang web hoặc ứng dụng của hãng hàng không
  * Tìm nhà ga trước khi đặt Uber đến sân bay [PHX]
  * Lưu ý về hạng vé phổ thông cơ bản
    * Không có hành lý xách tay miễn phí, chỉ được mang 1 vật dụng cá nhân ví dụ: ba lô Paradox
    * Làm thủ tục trực tuyến
    * Lên máy bay: bạn sẽ lên sau cùng
    * Ghế ngồi: được chỉ định khi làm thủ tục

## Địa điểm ăn uống và mua sắm

### Nhà hàng
* Pitch

>   * Địa điểm: [6350 E Thomas Rd, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/AwqDNMGFrp3HNVPx9)
>   * Thực đơn: cánh gà (hương thảo ngọt & cay, BBQ đào hun khói), burger, sandwich, pizza (pepperoni, Thái)
>   * Giá: $
>   * Đánh giá
>     * KHÔNG nên gọi Cánh Gà Cay Buffalo (hoặc cứ gọi và chuẩn bị trải nghiệm vị chua tột đỉnh của một món ăn)

Author: ```suggestion
  * Địa chỉ: [6350 E Thomas Rd, Scottsdale, AZ 85251, United States](https://goo.gl/maps/AwqDNMGFrp3HNVPx9)
  * Thực đơn: cánh gà (sốt hương thảo ngọt cay, sốt BBQ đào hun khói), burgers, sandwich, pizza (pepperoni, Thai pie)
  * Giá: $
  * Đánh giá
    * KHÔNG nên gọi món Buffalo Wings (Cánh gà sốt cay Buffalo), hoặc cứ gọi và chuẩn bị trải nghiệm vị chua đỉnh nóc kịch trần
```

* Saigon Bowl

>   * Địa điểm: [8123 E Roosevelt St, Scottsdale, AZ 85257, Hoa Kỳ](https://goo.gl/maps/WFbGoJL4FCKXEXPGA)

Author: ```suggestion
  * Địa chỉ: [8123 E Roosevelt St, Scottsdale, AZ 85257, United States](https://goo.gl/maps/WFbGoJL4FCKXEXPGA)
```

  * Thực đơn: các món ăn Việt Nam
  * Giá: $
  * Đánh giá
* RA Sushi Bar Restaurant

>   * Địa điểm: [3815 N Scottsdale Rd, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/yfnhbSLSyEg52vo6A)

Author: ```suggestion
  * Địa chỉ: [3815 N Scottsdale Rd, Scottsdale, AZ 85251, United States](https://goo.gl/maps/yfnhbSLSyEg52vo6A)
```

  * Thực đơn: các món ăn Nhật Bản
  * Giá: $
  * Đánh giá
* Craft 64 Scottsdale
  * Địa điểm: [6922 E Main St, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/gNyRt24Xii8XVxdW6)

>   * Thực đơn: Pizza, Bruschetta (bánh mì phô mai tỏi), Dolce (Tiramisu)
>   * Giá: $
>   * Đánh giá: Pizza mật ong là ngon nhất!

Author: ```suggestion
  * Thực đơn: Pizza, Bruschetta (bánh mì nướng phô mai tỏi), Dolce (Tiramisu)
  * Giá: $
  * Đánh giá: Pizza mật ong (honey bee pizza) là ngon nhất!
```

* Lo-Lo's Chicken and Waffles
  * Địa điểm: [3133 N Scottsdale Rd, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/if1HJkyG72Mm9aW17)
  * Thực đơn
  * Giá: $
  * Đánh giá: Tránh các món có hai bánh waffle, nếu không bạn sẽ phải xin hộp mang về.
* Phở Cao
  * Địa điểm: [7436 E McDowell Rd, Scottsdale, AZ 85257, Hoa Kỳ](https://goo.gl/maps/jcNXHixDSkjCHpUa9)
  * Thực đơn: các món ăn Việt Nam
  * Giá: $
  * Đánh giá:
    * Bên trong có thể rất ồn ào -- nếu bạn không thích nhạc, hãy ngồi ngoài
    * Có các món chay
* Sizzle Korean Barbecue
  * Địa điểm: [3720 N Scottsdale Rd #200, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/mLQaK1nGJADtpNim8)
  * Thực đơn: các món ăn Hàn Quốc
  * Giá: $$
  * Đánh giá
    * Nhân viên phục vụ sẽ nấu món ăn tại bàn của bạn, nhưng bạn có thể yêu cầu tự làm
    * Mì lạnh trộn sốt cay rất chua
* Hello Vietnam Pho & Roll

>   * Địa điểm: [2922 N Hayden Rd, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/WVuTRLf1135EM7Qa6)
>   * Thực đơn: các món ăn Việt Nam
>   * Giá: $
>   * Đánh giá
>     * Nếu Phở Cao/Saigon Bowl không hợp khẩu vị của bạn, hãy thử quán này.
>     * Lần trước tôi ăn ở đây, phở mặn quá phải xin thêm nước nóng 🙃 -- [@prd-khoa-luu](https://github.com/prd-khoa-luu)

Author: ```suggestion
  * Địa chỉ: [2922 N Hayden Rd, Scottsdale, AZ 85251, United States](https://goo.gl/maps/WVuTRLf1135EM7Qa6)
  * Thực đơn: các món ăn Việt Nam
  * Giá: $
  * Đánh giá
    * Thử quán này nếu Phở Cao/Saigon Bowl không hợp khẩu vị của bạn.
    * Lần trước ăn ở đây, phở mặn quá tôi phải xin thêm nước nóng 🙃 -- [@prd-khoa-luu](https://github.com/prd-khoa-luu)
```

* HULA'S Modern Tiki Scottsdale

>   * Địa điểm: [7213 E 1st Ave, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/8qehYP7LmXVbdbiSA)
>   * Thực đơn: các món ăn Mỹ/Polynesia/Hawaii
>   * Giá: $ - $$
>   * Đánh giá
>     * Có thể đông khách vào bữa trưa và bữa tối, vì vậy bạn có thể cần đặt chỗ trước.

Author: ```suggestion
  * Địa chỉ: [7213 E 1st Ave, Scottsdale, AZ 85251, United States](https://goo.gl/maps/8qehYP7LmXVbdbiSA)
  * Thực đơn: các món ăn Mỹ/Polynesia/Hawaii
  * Giá: $ - $$
  * Đánh giá
    * Quán có thể đông vào bữa trưa và bữa tối, vì vậy bạn có thể cần đặt chỗ trước.
```

* Schmooze
  * Địa điểm: [4222 N Marshall Way, Scottsdale, AZ 85251, Hoa Kỳ](https://goo.gl/maps/itDKT8iZkHccrvUK7)
  * Thực đơn: đồ uống cà phê & đồ ăn nhẹ cho bữa sáng & bữa trưa
  * Giá: $
  * Đánh giá
    * Thân thiện với công việc => Bạn có thể mang laptop đến đây
    * Khu vườn đẹp và mát mẻ
* Nhà hàng Da Vàng
  * Địa điểm: [4538 N 19th Ave, Phoenix, AZ 85015, Hoa Kỳ](https://maps.app.goo.gl/kB7SU7Brs3mK63CH7)
  * Thực đơn: các món ăn Việt Nam
  * Giá: $
  * Đánh giá
    * Nhà hàng Việt Nam 40 năm tuổi, khá xa, nhưng phở và bún bò Huế rất ngon
* Snakes & Lattes Tempe

>   * Địa điểm: [20 W 6th St, Tempe, AZ 85281, Hoa Kỳ](https://maps.app.goo.gl/s7BRJYc5JtXB4WV77)

Author: ```suggestion
  * Địa chỉ: [20 W 6th St, Tempe, AZ 85281, United States](https://maps.app.goo.gl/s7BRJYc5JtXB4WV77)
```

    * Thực phẩm đông lạnh và bữa ăn chế biến sẵn
    * Sản phẩm chăm sóc cá nhân
    * Thực phẩm bổ sung và thuốc không kê đơn (OTC)
    * Vật tư tẩy rửa và hàng gia dụng
    * Hàng theo mùa và quần áo cơ bản
    * Đồ điện tử, thiết bị gia dụng nhỏ và đồ dùng học tập
    * ... và đủ loại sản phẩm khác!
  * Các nhãn hiệu riêng bao gồm Great Value (hàng tạp hóa và gia dụng), Equate (sức khỏe và sắc đẹp), và Sam’s Choice (đồ uống và đồ ăn vặt)

* Costco
  * Yêu cầu thẻ thành viên (liên hệ với chúng tôi 😉)
  * Chỉ chấp nhận thẻ tín dụng/ghi nợ Visa
  * Không có túi, chỉ có hộp
  * 1 thành viên có thể mang theo tối đa 2 khách người lớn (cộng với bất kỳ trẻ em nào) mỗi lần ghé thăm, nhưng chỉ thành viên mới có thể mua hàng tại quầy thanh toán.
  * Giờ mở cửa: 10 giờ sáng – 8 giờ 30 tối (Thứ Hai – Thứ Sáu), 9 giờ 30 sáng – 6 giờ chiều (Thứ Bảy), 10 giờ sáng – 6 giờ chiều (Chủ Nhật); đóng cửa vào một số ngày lễ
  * Nên mua
    * Mua số lượng lớn (rẻ hơn)
    * Rau củ quả tươi: bông cải xanh, cải thìa, bắp cải, rau bina, dưa chuột, khoai tây, cà chua, cà rốt, măng tây
    * Trái cây: nho, chuối, táo, lê, dưa hấu
    * Thịt: bò, lợn, gà
    * Hải sản: cá, tôm

>     * Các mặt hàng bánh nướng: bánh ngọt, bánh quy, bánh ngọt (bánh danish nhân táo x2, hoặc bất kỳ **2** gói 4 chiếc nào), bánh quy xoắn, món tráng miệng

Author: ```suggestion
    * Các mặt hàng bánh nướng: bánh ngọt, bánh quy, bánh ngọt (bánh danish nhân táo x2, hoặc bất kỳ **2** lốc 4 cái nào), bánh quy xoắn, món tráng miệng
```

  * <img src="https://github.com/prd-khoa-luu/arizona-stay/assets/81603101/c0e6a866-1157-49e2-851a-c83ef9dde3d9" width="25%">

* Sở thú Phoenix
  * Địa điểm: [455 N Galvin Pkwy, Phoenix, AZ 85008, Hoa Kỳ](https://goo.gl/maps/aSJCmfMzTP1y5sry5)
  * Phí: $$
  * Có thể đặt vé trực tuyến - https://www.phoenixzoo.org/visit/
  * <img src="https://github.com/prd-khoa-luu/arizona-stay/assets/81603101/3a9da41a-9707-4445-9c6e-87840fc3f873" width="25%"> :flushed:

* Bảo tàng Ảo giác Scottsdale
  * Địa điểm: [9500 East Vía de Ventura, Scottsdale, AZ 85256, Hoa Kỳ](https://maps.app.goo.gl/sJK3akZta6RBCnFT6)
  * Phí: $
  * Có thể đặt vé trực tuyến - http://www.moiscottsdale.com/

* Tempe Marketplace
  * <img src="https://github.com/user-attachments/assets/c35d8f72-6939-4a82-b607-b42abeaa7f9e" width="25%"><img src="https://github.com/user-attachments/assets/fc0a62e1-ff0d-4961-8fb1-bc7f8ecda3b5" width="25%">
  * Địa điểm: [2000 E Rio Salado Pkwy, Tempe, AZ 85281, Hoa Kỳ](https://goo.gl/maps/jRm2TJf38BAqiARA9)
  * Trung tâm mua sắm ngoài trời với các nhà bán lẻ quốc gia, nhà hàng, nhạc sống vào cuối tuần & đài phun nước vui chơi cho trẻ em
  * Hoạt động chính

>     * Mua sắm: quần áo (Ross), đồ điện tử (Best Buy), mỹ phẩm (SEPHORA), kem dưỡng & xà phòng (Bath & Body Works), sách (Barnes & Noble)

Author: ```suggestion
    * Mua sắm: quần áo (Ross), đồ điện tử (Best Buy), mỹ phẩm (SEPHORA), sữa dưỡng thể & xà phòng (Bath & Body Works), sách (Barnes & Noble)
```

  * Kinh doanh & Giao thông
    * <img src="https://github.com/user-attachments/assets/1dd425b4-4ae6-41ce-8708-4618c333cf33" width="25%">
    * Văn phòng Công ty: Các công ty lớn và tòa nhà chính phủ
    * Trung tâm Giao thông: Tàu điện nhẹ kết nối với Tempe, Mesa và các khu vực xa hơn
    * <details><summary>Tàu điện nhẹ chạy qua Trung tâm Phoenix</summary><video src="https://github.com/user-attachments/assets/a0a1f009-7094-416b-9dbf-ea22585e942a" controls="controls"></video></details>

* Sedona
  * <details><summary>Trung tâm Sedona</summary><video src="https://github.com/user-attachments/assets/22fd2fcd-1d64-4380-b04a-469e50532b8c" controls="controls"></video></details>
  * Địa điểm: [Sedona, Arizona 86336](https://maps.app.goo.gl/xqKapGSrQttcN6EEA)

* Flagstaff
  * Địa điểm: [Flagstaff, Arizona, Hoa Kỳ](https://maps.app.goo.gl/iMthy7MTPeNwdomG6)

* Grand Canyon
  * <details><summary>Grand Canyon nhìn từ trên cao</summary><video src="https://github.com/user-attachments/assets/1f8c4432-645c-4226-9224-4dfc42381788" controls="controls"></video></details>
  * Địa điểm: [Vườn Quốc gia Grand Canyon, Arizona](https://maps.app.goo.gl/7uAAinDURpXBS1xf9)

* Ngoài Arizona

>   * Thành phố New York: Các địa danh mang tính biểu tượng (Tượng Nữ thần Tự do, Tòa nhà Empire State, Công viên Trung tâm, Quảng trường Thời đại), các bảo tàng đẳng cấp thế giới, các khu phố đa dạng (Khu phố Tàu, Harlem, Williamsburg)
>   * San Francisco: Cầu Cổng Vàng, các khu phố đa dạng (Khu phố Tàu, Khu Mission), Đảo Alcatraz, Bến Ngư phủ, xe cáp treo lịch sử, trụ sở và khuôn viên của các công ty công nghệ lớn (ở San Jose)
>   * Chicago: Kiến trúc nổi tiếng (Tháp Willis, các ngôi nhà của Frank Lloyd Wright), các điểm tham quan ven hồ (Bến tàu Navy, Công viên Millennium), và một khung cảnh nghệ thuật và sân khấu phát triển mạnh
>   * Seattle: Văn hóa cà phê (nơi khai sinh của Starbucks, các quán cà phê độc lập), Chợ Pike Place (hải sản tươi sống, cá bay), bờ sông tuyệt đẹp, Tháp Space Needle
>   * Boston: Lịch sử phong phú (Đường mòn Tự do, Nhà Paul Revere, Tàu Tiệc trà Boston), trung tâm thành phố dễ đi bộ, các trường đại học hàng đầu (Harvard, MIT), và các khu phố quyến rũ (Beacon Hill)
>   * Los Angeles: Các địa danh nổi tiếng thế giới (Biển hiệu Hollywood, Đại lộ Danh vọng Hollywood, Nhà hát Trung Hoa Grauman), các khu phố sôi động (Bãi biển Venice, Bến tàu Santa Monica, Beverly Hills cao cấp), các di tích lịch sử (Phố Olvera, Tháp Watts), Đài thiên văn Griffith

Author: ```suggestion
  * Thành phố New York: Các địa danh mang tính biểu tượng (Tượng Nữ thần Tự do, Tòa nhà Empire State, Công viên Trung tâm, Quảng trường Thời đại), bảo tàng đẳng cấp thế giới, các khu phố đa dạng (Chinatown, Harlem, Williamsburg)
  * San Francisco: Cầu Cổng Vàng, các khu phố đa dạng (Chinatown, Mission District), Đảo Alcatraz, Bến Ngư Phủ (Fisherman’s Wharf), xe cáp lịch sử, trụ sở và khuôn viên các công ty công nghệ lớn (ở San Jose)
  * Chicago: Kiến trúc nổi tiếng (Tháp Willis, nhà của Frank Lloyd Wright), các điểm tham quan ven hồ (Bến tàu Navy, Công viên Thiên niên kỷ), và nền nghệ thuật và sân khấu sôi động
  * Seattle: Văn hóa cà phê (nơi khai sinh Starbucks, các quán cà phê độc lập), Chợ Pike Place (hải sản tươi sống, cá bay), bờ sông đẹp như tranh vẽ, Tháp Space Needle
  * Boston: Lịch sử phong phú (Đường mòn Tự do - Freedom Trail, Nhà Paul Revere, Tàu và Bảo tàng Tiệc trà Boston), trung tâm thành phố dễ đi bộ, các trường đại học hàng đầu (Harvard, MIT), và các khu phố duyên dáng (Beacon Hill)
  * Los Angeles: Các địa danh nổi tiếng thế giới (Biển hiệu Hollywood, Đại lộ Danh vọng Hollywood, Nhà hát Trung Hoa Grauman), các khu phố sôi động (Bãi biển Venice, Bến tàu Santa Monica, khu Beverly Hills cao cấp), các địa điểm lịch sử (Phố Olvera, Tháp Watts), Đài thiên văn Griffith
```

## Phong tục/truyền thống/thói quen
* Ăn ngoài
  * Ở Mỹ, các nhà hàng thức ăn nhanh thường cho phép khách tự phục vụ đồ uống có ga miễn phí. Điều này có nghĩa là bạn thường có thể tự rót thêm đồ uống yêu thích nhiều lần mà không phải trả thêm phí.
  * Khi bạn vào một nhà hàng bình dân (như Applebee's, Olive Garden, hoặc Chili's), nhân viên phục vụ có thể hỏi bạn "dine-in or takeout?" (ăn tại chỗ hay mang đi?). Điều đó có nghĩa là họ đang hỏi bạn muốn ngồi xuống và thưởng thức bữa ăn tại nhà hàng ("dine in" hoặc "eat in") hay đặt món ăn để đóng gói và mang đi ăn ở nơi khác ("takeout" hoặc "takeaway" hoặc "to go").

>   * Khách hàng thường được kỳ vọng sẽ tự dọn khay và bỏ rác tại các chuỗi cửa hàng thức ăn nhanh (ví dụ: McDonald's, Chick-fil-A, Burger King, KFC, In-N-Out, Arby's, Wendy's)

Author: ```suggestion
  * Khách hàng cần tự dọn khay và bỏ rác của mình tại các chuỗi cửa hàng thức ăn nhanh (ví dụ: McDonald's, Chick-fil-A, Burger King, KFC, In-N-Out, Arby's, Wendy's)
```

* Tiền tip (boa)
  * Văn hóa tiền tip
    * Tiền tip không bắt buộc về mặt pháp lý, nhưng được xã hội kỳ vọng ở nhiều cơ sở dịch vụ, với mức 15-20% là tiêu chuẩn

>     * Nếu bạn không đủ khả năng tip, bạn không đủ khả năng đi ăn ngoài

Author: ```suggestion
    * Nếu bạn không đủ khả năng tip thì bạn đừng đi ăn ngoài
```

    * Nếu đồ ăn dở thì sao? --> Tiền tip là cho dịch vụ, không phải chất lượng đồ ăn; hãy cân nhắc tip bình thường và trình bày vấn đề đồ ăn với quản lý hoặc nhà bếp
    * Tip 10% hoặc ít hơn nếu dịch vụ thực sự tệ -- nhưng hiếm khi không tip gì trừ khi dịch vụ cực kỳ tồi tệ và bạn đã nói chuyện với quản lý
  * Số tiền
    * Nhà hàng có phục vụ tại bàn/dịch vụ đầy đủ: 15-20%, nhiều hơn cho dịch vụ xuất sắc
    * Dịch vụ tại quầy: 0-10%
    * Tại các chuỗi cửa hàng thức ăn nhanh lớn (xem ví dụ ở trên), tiền tip thường không được mong đợi hoặc yêu cầu
  * Phí phục vụ tự động (Automatic gratuities): 18% đến 20% hóa đơn
    * Thường được tự động thêm vào hóa đơn cho các nhóm lớn, với hơn 5 thực khách trong nhóm
    * Bạn không thể từ chối thanh toán phí phục vụ tự động vì đây là phí dịch vụ
    * Bạn có thể tip thêm vài đô la nếu dịch vụ tuyệt vời
  * Khi bạn thanh toán bằng thẻ tín dụng trong nhà hàng, quy trình diễn ra như sau

>     * Bạn yêu cầu nhân viên phục vụ mang hóa đơn (check/bill) - xem [Hội thoại cơ bản - Trong nhà hàng - Yêu cầu hóa đơn](#hoi-thoai-co-ban)

Author: ```suggestion
    * Bạn yêu cầu nhân viên phục vụ mang hóa đơn (check/bill) - xem [Hội thoại cơ bản - Trong nhà hàng - Yêu cầu hóa đơn](#hội-thoại-cơ-bản)
```

    * Bạn nhận hóa đơn, kiểm tra các món hàng, và đặt thẻ của bạn lên đó
    * Nhân viên phục vụ sẽ lấy thẻ của bạn và quay lại với một biên lai và một cây bút
    * Bạn viết số tiền tip, tính tổng, và ký tên
    * Đừng quên lấy biên lai và/hoặc chụp ảnh lại trước khi rời đi
    * <img src="https://github.com/prd-khoa-luu/arizona-stay/assets/81603101/2a8583cf-f906-4eb6-9c74-205e90e51b5c" width="50%">
  * Đối với các dịch vụ khác

>     * Thợ cắt tóc/làm tóc: 15–20% hóa đơn

Author: ```suggestion
    * Cắt tóc/làm tóc: 15–20% hóa đơn
```

    * Các gói hàng có giá trị cao có thể yêu cầu giấy tờ tùy thân

* Lái xe (không chắc có ai cần thông tin này không)
  * Bằng lái xe: không yêu cầu bằng lái xe quốc tế hoặc giấy phép, nhưng bạn phải mang theo bằng lái xe Việt Nam bên mình ([tại đây](https://azmvdservices.com/what-international-visitors-need-to-know-before-driving-in-arizona/), vui lòng tự kiểm tra lại thông tin này)
  * Thuê xe
    * Hầu hết các công ty cho thuê lớn yêu cầu người thuê xe phải từ 25 tuổi trở lên
    * Nếu bạn dưới 25 tuổi, bạn vẫn có thể thuê xe được, nhưng sẽ phải trả thêm phụ phí hàng ngày
  * Hãy chắc chắn rằng bạn hiểu
    * rằng luật pháp khác nhau, và phong tục lái xe cũng có thể khác nhau
    * phải làm gì nếu bạn gặp tai nạn xe hơi 😱

* Phép lịch sự khi đi Uber/taxi
  * Đặt xe
    * Hãy sẵn sàng rời đi trước khi yêu cầu xe. Đừng để tài xế phải đợi không cần thiết.
    * Nếu bạn dự đoán sẽ bị trễ, hãy thông báo cho tài xế qua ứng dụng hoặc gọi điện thoại.
    * Kiểm tra kỹ địa điểm đón của bạn. Sử dụng tính năng "ghi chú" nếu cần.
    * Hủy chuyến: Nếu bạn cần hủy chuyến, hãy hủy càng sớm càng tốt. Lưu ý rằng hủy quá muộn có thể bị tính phí.

>   * Chào hỏi & cảm ơn: xem [Hội thoại cơ bản - Trong xe Uber](#hoi-thoai-co-ban)

Author: ```suggestion
  * Chào hỏi & cảm ơn: xem [Hội thoại cơ bản - Trong xe Uber](#hội-thoại-cơ-bản)
```

| -------------------- | ----------------------- | ------------------------------- | ---------------------------------------- |
| Nhà hàng (ăn tại chỗ) | **+30%** | $100 + $10 thuế + $20 tip = $130 | Thuế + tip + có thể có phí thẻ          |
| Nhà hàng (mang đi)   | **+10-20%** | $100 + $10 thuế + $5 tip = $115  | Thuế + tip nhỏ (tùy chọn)             |
| Hàng tạp hóa         | **+2–3%** | $100 + $3 thuế = $103            | Một số thành phố đánh thuế hàng tạp hóa   |
| Mua sắm trực tuyến/Bán lẻ | **+9%** | $100 + $10 thuế = $110            | Thuế bán hàng kết hợp điển hình ở Arizona |

### Đơn vị & múi giờ
* Khoảng cách: 1 dặm (mi) ≈ 1.6 km
* Chiều dài
  * 1 foot (ft) ≈ 30 cm
  * 1 inch (in) ≈ 2.5 cm
* Trọng lượng
  * 1 pound (lb) ≈ 0.45 kg
  * 1 ounce (oz) ≈ 28 g
* Nhiệt độ: Độ Fahrenheit (°F)
  * Công thức: °F = (°C × 9/5) + 32
  * Mẹo nhanh
    * °F ≈ (°C × 2) + 30
    * °C ≈ (°F - 30) ÷ 2
    * 32°F ≈ 0°C
    * 50°F ≈ 10°C
    * 70°F ≈ 20°C
    * 90°F ≈ 30°C
    * 104°F ≈ 40°C
* Thời gian: Hầu hết Arizona luôn theo Giờ Chuẩn Miền Núi (MST, UTC-7), và không áp dụng [quy ước giờ mùa hè](https://en.wikipedia.org/wiki/Daylight_saving_time)

>   * Mùa đông (tháng 11 - tháng 3)
>     * Trước California 1 giờ (PST, UTC-8)
>     * Sau Chicago 1 giờ (CST, UTC−6)
>     * Sau New York 2 giờ (EST, UTC−5)
>   * Mùa hè (tháng 3 – tháng 11)
>     * Giống California (PDT, UTC-7)
>     * Sau Chicago 2 giờ (CDT, UTC-5)
>     * Sau New York 3 giờ (EDT, UTC−4)
>   * Arizona luôn sau Việt Nam 14 giờ (UTC+7)

Author: ```suggestion
  * Mùa đông (tháng 11 - tháng 3)
    * Sớm hơn California 1 giờ (PST, UTC-8)
    * Trễ hơn Chicago 1 giờ (CST, UTC−6)
    * Trễ hơn New York 2 giờ (EST, UTC−5)
  * Mùa hè (tháng 3 – tháng 11)
    * Giống California (PDT, UTC-7)
    * Trễ hơn Chicago 2 giờ (CDT, UTC-5)
    * Trễ hơn New York 3 giờ (EDT, UTC−4)
  * Arizona luôn trễ hơn Việt Nam 14 giờ (UTC+7)
```
